src/main/java/seedu/exercise/storage/resource/JsonAdaptedRegime.java

```diff
@@ -36,9 +36,9 @@ public JsonAdaptedRegime(@JsonProperty("name") String name,
     public JsonAdaptedRegime(Regime source) {
```

Comment:
javadoc comment for this should be `{@code Regime}` ?

Reply:
Changed the name to ``{@code Regime}``.